### PR TITLE
Updated studio docs

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

**Review by cooper-lzy — APPROVED** [review]

**Review by FengUyu — DISMISSED** [review]

File: docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md [hunk 1]
```diff
@@ -128,13 +128,15 @@ Nebula Graph Studio（ 以下简称 Studio ）支持云端或本地部署。云
 2. 使用`sudo rpm -i <rpm>`命令安装RPM包。
 
    例如，安装Studio {{ studio.base220 }} 版本需要运行以下命令：
+
    ```bash
    $ sudo rpm -i nebula-graph-studio-2.2.0-1.x86_64.rpm
```
Comment (whitewum) [inline docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md:133]: 220

File: docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md [hunk 2]
```diff
@@ -128,13 +128,15 @@ Nebula Graph Studio（ 以下简称 Studio ）支持云端或本地部署。云
...
    ```
 
 ### 卸载
 
 您可以使用以下的命令卸载 Studio。
+
 ```bash
 $ sudo rpm -e nebula-graph-studio-2.2.0-1.x86_64.rpm
```
Comment (whitewum) [inline docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md:141]: ditto

File: docs-2.0/nebula-studio/manage-schema/st-ug-crud-edge-type.md [hunk 3]
```diff
@@ -58,17 +58,20 @@ Studio v{{ studio.base220 }} 及以后版本。请更新版本，详细操作参
...
+  - 如果要修改 TTL：在 **设置TTL** 面板上，修改或 `TTL_COL` 和 `TTL_DURATION` 设置。
+  - 如果要删除所有已经配置的 TTL：在 **设置TTL** 面板的左上角，点击勾选框，取消选择。
+  - 如果要设置 TTL：在 **设置TTL** 面板的左上角，点击勾选框，开始设置 TTL。
 
 6. 完成设置后，在 **对应的nGQL语句** 面板上，您能看到修改后的 nGQL 语句。
 
 ## 删除边类型
 
+!!! danger
+    删除标签前先确认[影响](../../3.ngql-guide/10.tag-statements/2.drop-tag.md，否则可能造成数据损失。
```
Comment (whitewum) [inline docs-2.0/nebula-studio/manage-schema/st-ug-crud-edge-type.md:73]: ) missing

**Review by whitewum — COMMENTED** [review]
minor

**Review by cooper-lzy — APPROVED** [review]

File: docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md (hunk 1 above)
Comment (randomJoe211) [reply docs-2.0/nebula-studio/deploy-connect/st-ug-deploy.md:134]: The variable name defined in yaml is base220.

**Review by cooper-lzy — APPROVED** [review]

**Review by FengUyu — APPROVED** [review]

**Pull request merged**